products/nvidia-products.md

```diff
@@ -13,6 +13,41 @@ discontinuedColumn: true
 eoasColumn: true
 
 releases:
+-   releaseCycle: "mobile-professional-ada-lovelace"
+    releaseLabel: "Mobile Professional Blackwell (GB20x)"
+    releaseDate: 2025-03-18
+    eoas: false
+    eol: false
+    discontinued: false
+
+-   releaseCycle: "professional-blackwell"
+    releaseLabel: "Professional Blackwell (GB20x)"
+    releaseDate: 2025-03-17
+    eoas: false
+    eol: false
+    discontinued: false
+
+-   releaseCycle: "mobile-consumer-blackwell"
+    releaseLabel: "Mobile Consumer Blackwell (GB20x)"
+    releaseDate: 2025-03-01
```

Comment:
```suggestion
    releaseDate: 2025-01-30
    eoas: false
    eol: false
    discontinued: false

-   releaseCycle: "professional-blackwell"
    releaseLabel: "Professional Blackwell (GB20x)"
    releaseDate: 2025-01-30
    eoas: false
    eol: false
    discontinued: false

-   releaseCycle: "mobile-consumer-blackwell"
    releaseLabel: "Mobile Consumer Blackwell (GB20x)"
    releaseDate: 2025-01-30
```

On https://en.wikipedia.org/wiki/Blackwell_(microarchitecture)#Blackwell_dies, in the GB203 column, there are new models for consumer as well as workstation.